src/sass/Notes.sass

```diff
@@ -0,0 +1,52 @@
+.notes-container
+  height: 100%
+
+  .textarea
```

Comment:
Event better would be to give a special class to the note textarea and use it here instead. Because otherwise all `.textarea` in the `.notes-container` will have no styling, which could be surprising for another developper adding a textarea somewhere.